client/src/scene/SceneGameArena.ts

```diff
@@ -130,31 +130,31 @@ export class SceneGameArena extends Phaser.Scene {
         let moved = false;
         if (
             (this.keys.a.isDown || this.keys.left.isDown) &&
-            this.gameState.playerId !== null &&
+            this.gameState.playerId != null &&
```

Comment:
curious question: why we moved to equality from identity?

Reply:
A spectator does not have playerId set to null, but rather it is undefined. This check accounts for both situations where the playerId == null and when it's undefined.